src/scss/coral-talk-iframe/main.scss

```diff
@@ -65,13 +56,25 @@ a {
 	border-bottom-width: 0;
 }
 
+.coral-tabBarSecondary {
+	border-bottom-color: oColorsGetPaletteColor('teal-50');
+}
+
 
 // community guidelines box
 .coral-guidelines {
 	background-color: oColorsGetPaletteColor('wheat');
 	border-color: oColorsGetPaletteColor('wheat');
 }
 
+.coral-guidelines div div p {
+	letter-spacing: normal;
+}
+
+.coral-guidelines div div p a {
```

Comment:
Again I think this might be otherly specific. I think you might be able to achieve this with 

`.coral-guidelines a`